sdcm/cluster.py

```diff
@@ -1306,15 +1308,29 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         if not (self.is_rhel_like() or self.is_debian() or self.is_ubuntu()):
             raise ValueError('Unsupported Distribution type: {}'.format(str(self.distro)))
         if self.is_debian8():
+            self.remoter.run(cmd="sudo sed -i -e 's/jessie-updates/stable-updates/g' /etc/apt/sources.list")
+            self.remoter.run(cmd="sudo touch /etc/apt/sources.list.d/jessie-backports.list")
```

Comment:
There is a /etc/apt/sources.list.d/backports.list file exists in debian 8 instance of GCE.

So we need to process it, remove it or write our new jessi-backports repo to that file (replace it).
Otherwise, apt-get update will also fail for it.

Reply:
I don't understand...

Reply:
Here it touches/creates a new source file (`/etc/apt/sources.list.d/jessie-backports.list`), which has correct (archived) URL.

But in GCE debian8 instance, source file (`/etc/apt/sources.list.d/backports.list`) exists by default, it still uses wrong URL. This will causes the `apt-get update` fails.

So we need to replace `/etc/apt/sources.list.d/backports.list`, not create new `/etc/apt/sources.list.d/jessie-backports.list`

Reply:
@yarongilor you can reference my working (tested) workaround for debian 8:
- https://github.com/amoskong/scylla-cluster-tests/commit/52bab8d79a8084f54d94da3138a64bb9854ec3f5
- https://github.com/scylladb/scylla-cluster-tests/compare/master...amoskong:debian8

My workaround patch had been verified by upgrade job [link](http://jenkins.cloudius-systems.com:8080/job/scylla-3.0/job/3.0-test/job/scylla-rolling-upgrade-debian8-test/105/)

Reply:
i don't understand, does anyone test scylla-manager on GCE? i never tested it.

i think if we want to test manager on GCE we need a new PR and new testing environment for that.

i don't want to change code of something i never tested.

Reply:
My workaround also avoid problem in installing scylla, any test using debian8 of GCE as db node will has problem.

It's fine if you only want to fix something related with scylla-manager